src/cpp/core/system/PosixSystem.cpp

```diff
@@ -2572,10 +2572,6 @@ Error permanentlyDropPriv(const std::string& newUsername, const std::string& new
 // privilege manipulation for systems that don't support setresuid/getresuid
 #else
 
-namespace {
-   uid_t s_privUid;
```

Comment:
This appears to be used in Pro, so we need to be careful with the merge here.

Reply:
If that's true I'd try and keep the file the same between open-source and pro. But at a quick glance, I don't see where this gets used in Pro, either. Maybe it was accidentally left behind when things got split into shared_core? The one there does seem to be used:

https://github.com/rstudio/rstudio/blob/2715863751b3553cc5472db2f4532470c60300e4/src/cpp/shared_core/system/PosixSystem.cpp#L247